interfaces/acados_matlab/acados_template_mex/+acados_template_mex/generate_c_code_explicit_ode.m

```diff
@@ -0,0 +1,110 @@
+%   This file is part of acados.
```

Comment:
is there a reason to replicate all the `generate_c_code_xxx` files, instead of using the ones already present in the `acados_matlab` folder?

Reply:
I don't think so, it's WIP